FileUpload.py

```diff
@@ -0,0 +1,33 @@
+import jinja2
+import os
+import aiohttp_jinja2
+from zipfile import ZipFile
+from aiohttp import web
+
+routes = web.RouteTableDef()
+
+@routes.get('/')
+async def Homepage(request):
+    response = aiohttp_jinja2.render_template("index.html", request, context={})
+    return response
+
+@routes.post('/zip')
+async def UploadFile(request):
+    print("File Uploaded")
+    temp = await request.post()
+    get_file = temp['file'].file
+
+    with ZipFile(get_file) as file:
+        file_name = file.namelist()
+        print(file_name)
+        file.extractall(os.path.join(os.getcwd(),'save_file'))
```

Comment:
The files in the zip should be listed on the web page, and only required files should be downloaded from the list by clicking instead of extracting all